GTG/gtk/errorhandler.py

```diff
@@ -108,27 +108,31 @@ def _update_additional_info(self):
 
         text = ""
         if self._context_info is not None:
-            text = text + "Context: " + self._context_info + "\n\n"
-        text = text + body + "\n\n"
+            text = text + "**Context:** " + self._context_info + "\n\n"
+        text = text + "```\n" + body + "```\n\n"
```

Comment:
Maybe use

    ```python-traceback

so we have some colours to look at. Completely optional.

Reply:
I didn't know that was a thing! Very cool. Will do :)